### PR TITLE
added search bar to nav bar

### PR DESCRIPTION
**Pull request by Mebsworth**
@aharelick 
Added search bar. No actual logic - super basic

screen shot 2015-10-25 at 8 39 50 pm

### CODE REVIEW

File: app/templates/macros/nav_macros.html [hunk 1]
```diff
@@ -19,6 +19,9 @@
 
 {% macro account_items(current_user) %}
     {% if current_user.is_authenticated() %}
+        <div class="ui small inverted transparent icon input">
+            <input type="text" placeholder="Search members...">
```
Comment (aharelick) [inline app/templates/macros/nav_macros.html]: What are we planning on searching for here (members, resources)? I know it's just a placeholder, but I want to make sure the purpose is clear. Also, we should probably give it an id. Perhaps `id="search-nav"`.

**Comment by aharelick** [comment]
lgtm for now, feel free to merge

**Comment by Mebsworth** [comment]
sorry I think I might hold off on this - I have a better mockup that makes more sense with fitting everything on the nav bar

**Comment by skylerwharton** [comment]
@Mebsworth Can we close this pull request for now?

**Pull request closed without merging**